sdk/core/azure-mgmt-core/azure/mgmt/core/tools.py

```diff
@@ -217,3 +218,29 @@ def is_valid_resource_name(rname: str, exception_type: Optional[Type[BaseExcepti
     if exception_type:
         raise exception_type()
     return False
+
+
+def get_arm_info(cloud_setting: AzureClouds) -> Dict[str, Any]:
```

Comment:
```suggestion
def get_arm_info(cloud_setting: AzureClouds) -> Dict[str, str]:
```

to match the docstring

Reply:
actually maybe the docstring is wrong here as the return also has a list value

Reply:
Yes. I updated the return type but forgot to update the docstring.